pkgs/top-level/all-packages.nix

```diff
@@ -560,6 +560,8 @@ in
 
   antora = callPackage ../development/tools/documentation/antora {};
 
+  apfs-fuse-unstable = callPackage ../tools/filesystems/apfs-fuse/unstable.nix { };
```

Comment:
@ealasu seeing as there is only a single version of the software you should keep this as `apfs-fuse`.

Reply:
Done.